README.md

```diff
@@ -6,7 +6,7 @@
 [![Test Coverage](https://api.codeclimate.com/v1/badges/7a0b42c8a22c945571fd/test_coverage)](https://codeclimate.com/github/hashie/hashie/test_coverage)
 [![Maintainability](https://api.codeclimate.com/v1/badges/7a0b42c8a22c945571fd/maintainability)](https://codeclimate.com/github/hashie/hashie/maintainability)
 
```

Comment:
Huh...its still not rendering...You know what is especially strange about this... if you look at my forked repo branch here

https://github.com/carolineartz/hashie/tree/hashie-mascot#mascot

![image](https://user-images.githubusercontent.com/2850369/81433280-ec28b980-9129-11ea-8ac5-aef5f85b477b.png)

I feel like I have come across this before in forks but I just can't remember what I did about it 🤦‍♀️

Reply:
Works for me in https://github.com/hashie/hashie/blob/b781ad001de926710a287dc2b172d5b48b85ef30/README.md